compose/cli/main.py

```diff
@@ -268,6 +268,7 @@ def pull(self, project, options):
         Usage: pull [options] [SERVICE...]
 
         Options:
+            --skip-missing          Skip unpullable images.
```

Comment:
actually, I think we could be a little more informative here.  Maybe something like "Continue pulling images, even if one fails to pull".  

I'll let others comment on wording.